### PR TITLE
[C++/de] added.

### PR DESCRIPTION
**Pull request by LamdaLamdaLamda**
- [X ] I solemnly swear that this is all original content of which I am the original author
- [X] Pull request title is prepended with `[language/lang-code]`
- [X] Pull request touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [X] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by LamdaLamdaLamda** [comment]
@prertik fixed filename to the requested format.

File: de-de/c++-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,1160 @@
+---
+language: c++
+filename: learncpp.cpp
```
Comment (prertik) [inline de-de/c++-de.html.markdown]: Please keep language code in the filename.

**Comment by prertik** [comment]
Hmmm. There are two files introduced by the commits.

**Comment by prertik** [comment]
Cool. Thanks for your contribution.

**Pull request merged**